entry/src/main/js/MainAbility/pages/index/index.hml

```diff
@@ -0,0 +1,100 @@
+                </div>
+            </div>
+        </div>
+        <div class="middle">
+            <div class="caller" for="{{caller}}">
+                <div class="call">
+                    <image src="{{callericon}}" style=""></image>
+                    <text style="font-size: 32px; color: white; ;" onclick="onclickcaller"> {{$item.callerName}} </text>
+                    <image src="{{callaccept}}" style="margin-left: -2%;"></image>
+                    <image src="{{callreject}}" style="margin-left: -2%;"></image>
+                </div>
+                <text style="color: #8C8C8C; font-size: 14px; margin-left: 20%;">{{Incomingcall}}</text>
+            </div>
+            <div class="speedometer" for="{{speed}}">
+                <text style="color:white;font-size:90px;font-weight:800;margin-top:34.5px">{{$item.speedValue}}</text>
+                <text style="font-size:24px; color:#8C8C8C;margin-top:10.84px">{{$item.Kmh}}
+                </text>
+                <text style="font-size:18px; color:#8C8C8C ; margin-top:4.01px">{{avg}}
+                </text>
+                <text style="font-size:32px; color:white;margin-top:20.01px">{{$item.Kms}}
+                </text>
+            </div>
+        </div>
+        <div class="right" for="{{music}}">
+            <image src="{{$item.musicCover}}" style="height: 241px; width: 241px;"></image>
+            <text style="color: white; font-size: 18px; margin-top: 4%;">{{$item.AlbumName}}</text>
+            <text style="color: white; font-size: 14px; margin-top: 4%;">{{$item.musicName}}</text>
+            <div class="musicProgress">
+                <div>
+                    <progress class="musicbar" type="horizontal" percent= "70"></progress>
+                </div>
+                <div>
+                    <text style="color:#8C8C8C; ;margin-top: 5px; font-size:10px;color:white">{{music_start_timestamp}}</text>
+                    <text style="color:#8C8C8C; left: 600%;margin-top: 5px; font-size:10px;color:white">{{music_stop_timestamp}}</text>
+                </div>
+            </div>
+            <div class="musicButtons">
+                <image src="{{previousplay}}"  style=" width:28px;height:23px"></image>
+                <image src="{{pause}}" style=" width:22px;height:30px"></image>
+                <image src="{{nextplay}}" style=" width:52px;height:52px"></image>
+            </div>
+        </div>
+    </div>
+    <div class="child-container3">
+        <image src="{{menu}}"  onclick="menuclick"  style=" width:31px;height:31px"></image>
+        <image src="{{location}}" onclick="locationclick" style=" width:23px;height:30px"></image>
+        <image src="{{home}}"  style=" width:37px;height:37px;"></image>
+        <image src="{{headset}}"  onclick="hearphoneclick" style=" width:30px;height:30px"></image>
+        <image src="{{phone}}"  onclick="phoneclick" style=" width:30px;height:30px"></image>
+    </div>
+</div>
+
+
```

Comment:
1. .DS_Store many files are there, pls ask him to remove
2. code need to be formatted (css. js, hml files)
3. many emoty lines in end of index.hml file please remove

Reply:
But ma'am , code is already formatted using the keys  (Command + Alt + L) , as you mentioned in the previous meeting . Please , kindly recheck.